common/src/main/java/tachyon/Constants.java

```diff
@@ -139,8 +142,14 @@
   public static final String MASTER_RETRY_COUNT = "tachyon.master.retry";
 
   public static final String WORKER_MEMORY_SIZE = "tachyon.worker.memory.size";
+  public static final String WORKER_HOSTNAME = "tachyon.worker.hostname";
+  public static final String WORKER_BIND_HOST = "tachyon.worker.bind.host";
   public static final String WORKER_PORT = "tachyon.worker.port";
+  public static final String WORKER_DATA_HOSTNAME = "tachyon.worker.data.hostname";
+  public static final String WORKER_DATA_BIND_HOST = "tachyon.worker.data.bind.host";
   public static final String WORKER_DATA_PORT = "tachyon.worker.data.port";
+  public static final String WORKER_WEB_HOSTNAME = "tachyon.worker.web.hostname";
+  public static final String WORKER_WEB_BIND_HOST = "tachyon.worker.web.bind.host";
```

Comment:
The followings are several examples for the multihomed networks:
Spark: https://issues.apache.org/jira/browse/SPARK-5719
YARN: https://issues.apache.org/jira/browse/YARN-1994
Hadoop: https://issues.apache.org/jira/browse/HADOOP-8198
MapReduce: https://issues.apache.org/jira/browse/MAPREDUCE-4168